dist/vidBg.js

```diff
@@ -120,7 +120,7 @@ module.run(['$templateCache', function($templateCache) {
     '		<source type="video/webm">\n' +
     '		<source type="video/mp4">\n' +
     '		<source type="video/ogg">\n' +
-    '		<object type="application/x-shockwave-flash" data="{{resourceMap.flash}}">\n' +
+    '		<object type="application/x-shockwave-flash" ng-data="{{resourceMap.swf}}">\n' +
```

Comment:
I was actually copying over this part from some other sources. Have you tested your changes for flash files?